packages/circuit-ui/components/Table/Table.docs.mdx

```diff
@@ -36,6 +36,23 @@ While headers and rows can accept strings as children, they can also accept obje
 
 <Story id="components-table--with-component-rows" />
 
+### Expandable
+
+Add child elements to your data to have expandable rows:
```

Comment:
Nice, I like the API for it. It seems consistent with the rest of it and should be a nonbreaking change 👍